internal/policy/verify.go

```diff
@@ -656,35 +669,18 @@ func (v *Verifier) Threshold() int {
 // signature and signatures embedded in a DSSE envelope. Verify does not inspect
 // the envelope's payload, but instead only verifies the signatures. The caller
 // must ensure the validity of the envelope's contents.
-func (v *Verifier) Verify(ctx context.Context, gitObjectID gitinterface.Hash, env *sslibdsse.Envelope) error {
+func (v *Verifier) Verify(ctx context.Context, gitObjectID gitinterface.Hash, authorizationAttestationEnv, githubApprovalAttestationEnv *sslibdsse.Envelope) error {
```

Comment:
One big question in my mind is that this interface is going from caring strictly about a git sig and dsse sigs to being aware of a specific predicate type. I think we may want to revert this change and handle the special casing of the github approval attestations separately.

Reply:
working on rolling this back

Reply:
i've pushed changes on this, i'll clean up tomorrow.